src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -70,13 +70,15 @@ protected void onCreate(Bundle state, boolean ready) {
     webSettings.setGeolocationEnabled(false);
     webSettings.setAllowFileAccessFromFileURLs(false);
     webSettings.setAllowUniversalAccessFromFileURLs(false);
+    webSettings.setDatabaseEnabled(true);
+    webSettings.setDomStorageEnabled(true);
     webView.addJavascriptInterface(new InternalJSApi(), "InternalJSApi");
 
     // `msg_id` in the subdomain makes sure, different apps using same files do not share the same cache entry
     // (WebView may use a global cache shared across objects).
     // (a random-id would also work, but would need maintenance and does not add benefits as we regard the file-part interceptRequest() only,
     // also a random-id is not that useful for debugging)
-    webView.loadUrl("webxdc://msg" + appMessageId + ".localhost/index.html");
+    webView.loadUrl("https://msg" + appMessageId + ".localhost/index.html");
```

Comment:
do we need https for this on android?
on desktop I had to define the webxdc scheme as special "priviliged scheme" to get it working, so could be that this is required.
anyways its an implementation detail so not that important as long as it works and because we (can) intercept all requests on android using `https` works.

Reply:
yes, `https` is needed on android, check comments in: https://github.com/deltachat/deltachat-android/pull/2221/commits/3df96825aefefc3d55da76b3b1c7be51d9ace612